### ISSUE TITLE
Mada network is not there in supported networks

### ISSUE THREAD

**Issue by AnilBattaram-Feker** [busfor/react-native-apay#17]
Mada network is required for payments in middle east

**Comment by sorokin0andrey** [comment]
Hi! Could you test it if i support?

**Comment by AnilBattaram-Feker** [comment]
Yes. We will test it. At present, we are making below changes in
react-native-apay  module before releasing mobile app build.

(After doing npm install, in react-native-apay, we are doing the following
changes to include mada network and releasing mobile build.)

(react-native-apay/ios/RNApplePay.m)

if ***@***.***(iOS 12.0, *)) {

[supportedNetworksMapping setObject:PKPaymentNetworkMaestro forKey:@
"maestro"];

*[supportedNetworksMapping setObject:PKPaymentNetworkMada forKey:@"mada"];*

}


[image: image.png]



(react-native-apay/index.d.ts)

Add "mada" in APayAllowedCardNetworkType (first line which is supported
cards)
export type APayAllowedCardNetworkType = "amex" | "mastercard"| "visa" |
"privatelabel" | "chinaunionpay" | "interac" | "jcb" | "suica" |
"cartebancaires" | "idcredit" | "quicpay" | "maestro"* | "mada"*

Regards,
Anil
